docs/datadog_dashboard.md

```diff
@@ -0,0 +1,82 @@
+# Datadog Dashboards
+This feature will be introduced in DatadogOperator v1.9.0 and is currently in beta.
+
+## Overview
+The Datadog Dashboards CRD allows users to create dashboards using the operator and manage them as Kubernetes resources. 
+
+All configurable Dashboard fields are available in the CRD: `Description`, `LayoutType`, `NotifyList`, `ReflowType`, `Tags`, `TemplateVariables`, `TemplateVariablePresets`, `Title`, and `Widgets`. (https://docs.datadoghq.com/dashboards/)
+
+There are two main ways to define the Spec of a Datadog Dashboard. The first way is by specifying each field in a yaml file like so:
+```
+apiVersion: datadoghq.com/v1alpha1
+kind: DatadogDashboard
+metadata:
+  name: example-dashboard
+  namespace: system 
+spec:
+  title: metricsTest
+  layoutType: ordered
+
+  widgets:
+    - timeseries:
+        title: "itsTime"
+        titleAlign: center
+        titleSize: "250"
+        requests: 
+          - queries:
+              - metricQuery: 
+                  query: avg:system.cpu.user{*}
+                  name: metrics
+                  dataSource: metrics
+                  aggregator: avg
+            responseFormat: timeseries
+            style:
+              lineType: dotted
+              lineWidth: thin
+            onRightYaxis: false
+      id: 124
+```
+However, because adding widgets is a very big pull, only two widget types (`Timeseries` and `QueryValue`) are supported this way.
+
+The second way is by copying the JSON of a dashboard in app, escaping it and just putting that into the `Data` field. Since this string isn't as easy to configure, there
```

Comment:
```suggestion
The second way is by copying the JSON of a dashboard in app, escaping it, and putting that into the `Data` field. Since this string isn't as easy to configure, there
```